frontend/src/lib/get-cluster.ts

```diff
@@ -38,6 +38,7 @@ export enum ClusterStatus {
     'posthookjob' = 'posthookjob',
     'posthookfailed' = 'posthookfailed',
     'importfailed' = 'importfailed',
+    'nojobs' = 'nojobs',
```

Comment:
This doesn't look needed when I look at the changes

Reply:
Brain malfunctioned here, thanks.